### PR TITLE
fix: add retries to test

### PR DESCRIPTION
**Pull request by averikitsch**
fixes: #2312

### CODE REVIEW

File: run/websockets/test/system.test.js [hunk 1]
```diff
@@ -123,12 +123,23 @@ describe('End-to-End Tests', () => {
...
-    const list = await browserPage.$('#messages li');
-    const itemText = await browserPage.evaluate(el => el.textContent, list);
-    assert.strictEqual(itemText.trim(), 'Sundar: Welcome!');
+    let itemText;
+    for (let i = 0; i < 5; i++) {
+      itemText = await browserPage.evaluate(() => {
+        return document.querySelector('#messages li');
+      });
+      if (itemText) return;
+      sleep(i * 1000); // Linear delay
```
Comment (bcoe) [inline run/websockets/test/system.test.js:132]: I think  you want to `await sleep(i * 1000);` here, otherwise the loop won't block.

Comment (averikitsch) [reply run/websockets/test/system.test.js]: Oops. Thanks!

**Comment by averikitsch** [comment]
ping @bcoe

**Review by bcoe — APPROVED** [review]

**Comment by bcoe** [comment]
@averikitsch you may need to push an empty commit at this branch `git commit --allow-empty -m "force build"` there was an issue with actions earlier this week.

**Pull request merged**